README.md

```diff
@@ -1,71 +1,71 @@
-Riot
-====
+Element
+=======
 
-Riot (formerly known as Vector) is a Matrix web client built using the [Matrix
+Riot (formerly known as Vector and Riot) is a Matrix web client built using the [Matrix
```

Comment:
```suggestion
Element (formerly known as Vector and Riot) is a Matrix web client built using the [Matrix
```

Reply:
I'm like 99% sure I changed this

Reply:
Oh, that's weird lol. Not sure how I missed this...